class-wc-calypso-bridge.php

```diff
@@ -172,7 +174,9 @@ public function load_ui_elements() {
 			add_action( 'admin_enqueue_scripts', array( $this, 'add_ecommerce_plan_styles' ) );
 
 			// Nav unification fixes.
-			if ( function_exists( 'wpcomsh_activate_nav_unification' ) && wpcomsh_activate_nav_unification( false ) ) {
+		if ( function_exists( 'wpcomsh_activate_nav_unification' )
```

Comment:
Is the indentation off here?

Reply:
Thanks for catching this! I've fixed indentation in 692e24c16cee7f99ca392375de45c9877dd0b344